.gitleaks.toml

```
@@ -4,4 +4,5 @@
   paths = [
     # Ignore all testdata cert keys.
     '''\/testdata.*\.key$''',
+    '''values-private.*\.yaml\.example$''',
```

Is this intended change?

---

When I ran `git merge master` into this branch, my RH pre-commit hook flagged a bunch of these [example value files](https://github.com/stackrox/stackrox/blob/49ab6258de11d4f5e960b1d2bc44a8084f528d6b/image/templates/helm/stackrox-central/values-private.yaml.example#L38) in the helm chart.  This change allowed my merge to be committed.

I suspect why this hasn't been hit before is because I had to resolve a merge conflict after running `git merge master`. After the initial merge command, I addressed the conflict, added the files, and then ran `git merge --continue`, which is when I got the pre-commit errors.

---

I can separate this out into a different PR if you'd like.

---

No, it's fine to keep. I'd suggest to add a comment around that line to show intention, and maybe also scope path (untested):
```suggestion
    # Example value files in Helm charts.
    '''stackrox/image/templates/helm/.*values-private.*\.yaml\.example$''',
```

---

After failing to reproduce an error by the pre-commit hook, I've just removed this line altogether.  Perhaps there is some kind of bug with the pre-commit hook?  Or maybe there is a problem where it should be flagging those files but it's normally not?  Regardless, removing the line does not cause any issues for me at this point.